components-api/src/main/scala/pl/touk/nussknacker/engine/api/context/ValidationContext.scala

```diff
@@ -13,6 +13,17 @@ import javax.lang.model.SourceVersion
 object ValidationContext {
 
   def empty: ValidationContext = ValidationContext()
+
+  def isVariableNameValid(name: String): Boolean = SourceVersion.isIdentifier(name)
+
+  def validateVariableName(name: String, paramName: Option[String])(
```

Comment:
is this the best place for these methods? they don't depend on the Context at all
maybe just create a separate object for them

Reply:
Yeah, I think that we won't find better.